src/main/java/net/blay09/mods/waystones/client/WaystoneCreativeTab.java

```diff
@@ -0,0 +1,21 @@
+package net.blay09.mods.waystones.client;
+
+import net.blay09.mods.waystones.Waystones;
+import net.minecraft.item.ItemStack;
+
+public class WaystoneCreativeTab extends net.minecraft.creativetab.CreativeTabs {
```

Comment:
Also should add CreativeTabs to the imports rather than using a fully qualified name